hail/python/hail/docs/hailpedia/matrix_table.rst

```diff
@@ -293,15 +296,15 @@ a Python value.
 
 An example of querying entries is to compute the global mean of field `GQ`:
 
-    >>> mt.aggregate_entries(hl.agg.mean(mt.GQ))
+    >>> mt.aggregate_entries(hl.agg.mean(mt.GQ))  # doctest: +NOTEST
     67.73196915777027
```

Comment:
is this too expensive or something?

Reply:
er, should be fine. I think if anything it was the floating point precision changing run to run

Reply:
but the cost doesn't change -- NOTEST means run the thing, but don't test the output